simcon_project/templates/researcher_base.html

```diff
@@ -23,6 +23,7 @@
         </ul>
         <ul class="nav navbar-nav navbar-right">
           <li>{% block nav-userView %}<a href="{% url 'researcher-user-view' %}"><span class="glyphicon glyphicon-user"></span> User</a>{% endblock %}</li>
+          <li>{% block nav-logoutView %}<a href="{% url 'logout' %}"><span class="	glyphicon glyphicon-log-out"></span> Log Out</a>{% endblock %}</li>
```

Comment:
I'd like logout to be the right-most thing on the nav-bar. 

Other opinions?

Reply:
Will Adjust!